Plugins/OpenAPIGeneratorCommand/plugin.swift

```diff
@@ -65,27 +65,39 @@ extension SwiftOpenAPIGeneratorPlugin: CommandPlugin {
         var hadASuccessfulRun = false
 
         for target in targets {
-            print("Considering target '\(target.name)':")
+            log("Considering target '\(target.name)':")
             guard let swiftTarget = target as? SwiftSourceModuleTarget else {
-                print("- Not a swift source module. Can't generate OpenAPI code.")
+                log("- Not a swift source module. Can't generate OpenAPI code.")
                 continue
             }
             do {
-                print("- Trying OpenAPI code generation.")
+                log("- Trying OpenAPI code generation.")
                 try runCommand(
                     targetWorkingDirectory: target.directory,
                     tool: context.tool,
                     sourceFiles: swiftTarget.sourceFiles,
                     targetName: target.name
                 )
-                print("- ✅ OpenAPI code generation for target '\(target.name)' successfully completed.")
+                log("- ✅ OpenAPI code generation for target '\(target.name)' successfully completed.")
                 hadASuccessfulRun = true
             } catch let error as PluginError {
+                if targetNameArguments.isEmpty, case .fileErrors(let errors) = error,
+                    errors.map(\.fileKind) == FileError.Kind.allCases,
```

Comment:
```suggestion
                    Set(errors.map(\.fileKind)) == Set(FileError.Kind.allCases),
```

This relies on the fact that we first check the config file, then the document, and that matches the order in allCases. If you'd like to make it more robust by checking the contents as sets, you can accept this proposed change - but it's also fine if you don't.